### DISCUSSION TITLE
Add support for Trezor Model T on Android devices

### DISCUSSION THREAD

**Discussion by mark-stopka** [Emurgo/yoroi-mobile#1877] [Ideas]
USB-C enabled mobile phones with [Android can support Trezor Model T](https://wiki.trezor.io/User_manual:Using_Trezor_with_Android), it would be great it Android version of Yoroi would have support for Trezor just like it has for Ledger.

**Comment by SebastienGllmt** [comment]
This would be a good Catalyst proposal if somebody wants to build this functionality

**Comment by mark-stopka** [comment]
> This would be a good Catalyst proposal if somebody wants to build this functionality

Maybe we could add a label "Help Needed/Catalyst Idea"?

**Comment by mark-stopka** [comment]
> I think you should write to the Yoroi wallet support team for Help. They are fast and Reliable. {link removed}

Beware, this is a scam...

**Comment by mark-stopka** [comment]
@rcmorano I was wondering if you put it under consideration?

**Reply by rcmorano** [reply]
> @rcmorano I was wondering if you put it under consideration?

hey Mark! I'm not part of Yoroi team since some time ago ^^ But if you ask me, I would love to have Trezor T working with my Cardano mobile of choice :>

**Reply by mark-stopka** [reply]
> > @rcmorano I was wondering if you put it under consideration?
> 
> hey Mark! I'm not part of Yoroi team since some time ago ^^ But if you ask me, I would love to have Trezor T working with my Cardano mobile of choice :>

Is this private or public debate? Either way, I do think that Cardano phone and Cardano Desktop OS is a way to go, so if there is anything I can do to be part of that conversation, I am all ears =)

I have been involved in open-source hardware community for some quite time, so I am more than happy to help out if we communicate clearly, and if you wish under NDA.

**Reply by mark-stopka** [reply]
@rcmorano [PinePhone](https://www.pine64.org/pinephone/) running [openSUSE Cardano Edition](https://code.opensuse.org/leap/features/issue/59) perhaps?

**Reply by rcmorano** [reply]
WOOHOO! Are there OBS builds for those already :smirk:?

**Reply by rcmorano** [reply]
> WOOHOO! Are there OBS builds for those already smirk?

nvm, I've just went thru the thread you shared and catched up with the state-of-the-art :)) Awesome job on the openSUSE side Mark, love it!
